src/components/Section/Citizenship/Status/Status.jsx

```diff
@@ -76,12 +79,47 @@ export class Status extends Subsection {
     })
   }
 
-  updateField = (field, values) =>  {
+  updateField = (field, values) => {
     this.update({
       [field]: values,
     })
   }
 
+  isRequired = () => {
```

Comment:
This logic feels like it should live somewhere so it can be shared across the validation function & this component